runtime/westend/src/lib.rs

```diff
@@ -82,7 +82,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("westend"),
 	impl_name: create_runtime_str!("parity-westend"),
 	authoring_version: 2,
-	spec_version: 2000,
+	spec_version: 20,
```

Comment:
The idea behind this was from @kirushik for making it easier to detect the addition of `transaction_version`.